cmd/plasmacli/subcmd/eth/query/confirmsig.go

```diff
@@ -0,0 +1,91 @@
+package query
+
+import (
+	"encoding/json"
+	"fmt"
+	"github.com/FourthState/plasma-mvp-sidechain/cmd/plasmacli/config"
+	"github.com/FourthState/plasma-mvp-sidechain/plasma"
+	"github.com/FourthState/plasma-mvp-sidechain/store"
+	"github.com/cosmos/cosmos-sdk/client/context"
+	"github.com/ethereum/go-ethereum/rlp"
+	"github.com/spf13/cobra"
+)
+
+// SigCmd returns the query confirm sig command
+func SigCmd() *cobra.Command {
+	config.AddPersistentTMFlags(sigCmd)
+	sigCmd.Flags().Bool(useNodeF, false, "trust connected full node")
+	return sigCmd
+}
+
+var sigCmd = &cobra.Command{
+	Use:   "sig <position>",
+	Short: "Query confirm signature information for a given position",
+	Args:  cobra.ExactArgs(1),
+	RunE: func(cmd *cobra.Command, args []string) error {
+		// parse position
+		pos, err := plasma.FromPositionString(args[0])
+		if err != nil {
+			return err
+		}
+
+		var sigs []byte
+		ctx := context.NewCLIContext()
+		sigs, err = getSigs(ctx, pos)
+		if err != nil {
+			return fmt.Errorf("failed to retrieve confirm signature information: %s", err)
+		}
+
+		switch len(sigs) {
+		case 65:
+			fmt.Printf("Confirmation Signatures: 0x%x\n", sigs[:])
+		case 130:
+			fmt.Printf("Confirmation Signatures: 0x%x, 0x%x\n", sigs[:65], sigs[65:])
+		}
+
+		return nil
+	},
+}
+
+// Returns confirm sig results for given position
+// Trusts connected full node
+func getSigs(ctx context.CLIContext, position plasma.Position) ([]byte, error) {
```

Comment:
This should go in [client/query.go](https://github.com/FourthState/plasma-mvp-sidechain/blob/master/client/query.go), right @hamdiallam ?

Reply:
@hamdiallam

Reply:
Yeah, this should be in client module so that it can be reused outside of the command line (i.e in a rest handler).